### DISCUSSION TITLE
All admin -routes/pages are exposed in source code

### DISCUSSION THREAD

**Discussion by ryggdunk** [filamentphp/filament#6587] [Help] [answered]
I use Laravel Breeze with Inertia and Vue set up.
If I go to any (non Filament-admin) page and click View Page Source, I can see all my Filament resources/pages/routes for some reason inside a javascript 'const Ziggy'.

Is this how it's supposed to be or maybe I've done something wrong? In any case it feels like a big vulnerability. I rather that information not be seen to anyone who isn't logged in.

```
 <script type="text/javascript">
 const Ziggy = {"url":"https:\/\/mywebsite.com","port":null,"defaults":{},"routes":{"filament.asset":{"uri":"filament\/assets\/{file}","methods":["GET","HEAD"],"wheres":{"file":".*"}},"filament.auth.logout":{"uri":"filament\/logout","methods":["POST"]},"filament.auth.login":{"uri":"admin\/login","methods":["GET","HEAD"]},"filament.pages.dashboard":{"uri":"admin","methods":["GET","HEAD"]},"filament.resources.articles.index":{"uri":"admin\/articles","methods":["GET","HEAD"]},"filament.resources.articles.create":{"uri":"admin\/articles\/create","methods":["GET","HEAD"]},"filament.resources.articles.edit":
...
</script>
```

Any ideas?

**Comment by danharrin** [comment, marked as answer]
I don't think thats our problem tbh, that is just the way Ziggy works, otherwise it wouldn't be able to generate URLs for those routes. I dont think its a security concern unless you neglect authorization.

**Reply by ryggdunk** [reply]
Yeah, I figured it has nothing to do with Filament. But I decided to skip Inertia and separate the admin -panel and the main app so it won't be problem hopefully.
As for security, even if they can't use those routes without authorization it still gives people a lot of insight to what kind of functionality I'm using in my admin just based on the name of those routes. In my case that's not acceptable.
